kinde_sdk/kinde_api_client.py

```
@@ -335,32 +336,26 @@ def _decode_token_if_needed(self, token_name: str) -> None:
+                "algorithms":["RS256"],
+                "options":{
+                    "verify_signature": True,
+                    "verify_exp": True,
+                    "verify_aud": False
                 }
-                return jwt.decode(**decode_token_params)
-            else:
-                raise KindeTokenException(f"Token {token_name} doesn't exist.")
-        return token_value
+            }
+            return {token_name: jwt.decode(**decode_token_params)};
```

Ruff (0.8.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)